src/mappings/Cache.ts

```diff
@@ -34,7 +34,10 @@ export async function cachedRewardDestination(accountAddress: string, event: Sub
             return await api.query.staking.payee(accountAddress)
         }
 
-        const payees = await api.query.staking.payee.multi(allAccountsInBlock);
+        // TODO: Commented code doesn't work now, may be fixed later
```

Comment:
right, probably it will be a good idea to deploy only transfers logic for chains that doesn't have native staking. It will also be indexed faster